provider/spark_test.go

```diff
@@ -54,17 +55,76 @@ func testMaterializeResource(store *SparkOfflineStore) error {
 		return fmt.Errorf("Did not properly register table")
 	}
 
+	matID := MaterializationID("Materialization/test_name_materialize/test_variant")
```

Comment:
I would set the string `"Materialization/test_name_materialize/test_variant"` into a variable and reference the variable later on instead of hardcoding it.

Reply:
changed